samples/charts/data-chart/axis-crossing/App.razor

```diff
@@ -0,0 +1,61 @@
+﻿@page "/DataChartAxisAnnotations"
+
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igComponent">        
+        @if (SinData != null && CosData != null)
+        {
+        <DataChart Height="100%" Width="100%" IsVerticalZoomEnabled="true" IsHorizontalZoomEnabled="true">
+            <NumericXAxis Name="xAxis" Interval="20" CrossingAxisName="yAxis" CrossingValue="0" MinimumValue="0" MaximumValue="360" />
+            <NumericYAxis Name="yAxis" LabelLocation="@AxisLabelsLocation.OutsideRight" MinimumValue="-1.25" MaximumValue="1.25" Interval="0.25" />
+
```

Comment:
y-axis should have property settings for
CrossingAxisName="xAxis" CrossingValue="0"
LabelLocation="AxisLabelsLocation.InsideLeft"
x-axis should have property settings for
LabelLocation="AxisLabelsLocation.InsideBottom"